### DISCUSSION TITLE
"LARP" stands for Live Action "What" Playing?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#462] [Galaxy Trivia] [answered]
**Bonus:** What does this term mean?

**Comment by peterbaccaro** [comment, marked as answer]
LARP stands for **live action role-playing game**.  It is a form of role-playing game where the participants physically portray their characters. The players pursue goals within a fictional setting represented by real-world environments while interacting with each other in character.
